### PR TITLE
Tf model

### PR DESCRIPTION
**Pull request by bellycat77**
Object detection from [https://github.com/tensorflow/models/](https://github.com/tensorflow/models/) was added

### CODE REVIEW

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "install_env": {
+      "PACKAGE_NAME": "faster_rcnn_resnet101_kitti_2018_01_28.tar.gz",
+      "PACKAGE_NAME1": "faster_rcnn_resnet101_kitti_2018_01_28",
+      "FROZEN_GRAPH": "frozen_inference_graph.pb",
+      "PACKAGE_URL": "http://download.tensorflow.org/models/object_detection",
+      "MODEL_WEIGHTS_ARE_CHECKPOINTS": "YES",
+      "WEIGHTS_FILE": "model.ckpt",
+      "MODULE_FILE": "mobilenet-model.py",
+      "DATASET_NAME": "kitti",
+      "MODEL_NAME": "Faster RCNN resnet101 with Kitti dataset",
```
Comment (psyhtest) [inline package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json]: Faster-RCNN-ResNet101 for KITTI

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      "local": "yes", 
+      "name": "TensorFlow Library", 
+      "tags": "lib,tensorflow"
+    }, 
+    "tensorflow-model-api": {
+      "local": "yes", 
+      "name": "Tensorflow models api", 
+      "tags": "tensorflow,model,api"
+    }
+
```
Comment (psyhtest) [inline package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json]: Extra blank line.

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  }, 
+  "deps": {
+    "tensorflow": {
+      "local": "yes", 
+      "name": "TensorFlow Library", 
+      "tags": "lib,tensorflow"
+    }, 
+    "tensorflow-model-api": {
+      "local": "yes", 
+      "name": "Tensorflow models api",
```
Comment (psyhtest) [inline package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json]: TensorFlow Object Detection API?

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  "end_full_path": {
+    "linux": "model/faster_rcnn_resnet101_kitti.config"
+  }, 
+  "only_for_host_os_tags": [
+    "linux"
+  ], 
+  "only_for_target_os_tags": [
+    "linux"
+  ], 
+  "package_extra_name": " (Faster RCNN resnet101, kitti)",
```
Comment (psyhtest) [inline package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json]: Faster-RCNN-ResNet101, KITTI

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/scripts.linux/install.sh [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "Copy classes' labels ..."
```
Comment (psyhtest) [inline package/tensorflowapimodel-faster-rcnn-resnet101-kitti/scripts.linux/install.sh]: "Copy class labels"?

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/kitti_label_map.pbtxt [hunk 6]
```diff
@@ -0,0 +1,9 @@
+item {
+  id: 1
+  name: 'car'
+}
+
+item {
+  id: 2
+  name: 'pedestrian'
+}
```
Comment (psyhtest) [inline package/tensorflowapimodel-faster-rcnn-resnet101-kitti/kitti_label_map.pbtxt]: Hmm, no 'cyclists'?

File: package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json [hunk 7]
```diff
@@ -3,52 +3,63 @@
...
-    "no_ver_in_suggested_path": "yes",
+      "PACKAGE_NAME": "ssd_mobilenet_v1_coco_2018_01_28.tar.gz",
+      "PACKAGE_NAME1": "ssd_mobilenet_v1_coco_2018_01_28",
+      "FROZEN_GRAPH": "frozen_inference_graph.pb",
+      "PACKAGE_URL": "http://download.tensorflow.org/models/object_detection",
+      "MODEL_WEIGHTS_ARE_CHECKPOINTS": "YES",
+      "WEIGHTS_FILE": "model.ckpt",
+      "MODULE_FILE": "mobilenet-model.py",
+      "DATASET_NAME": "coco",
+      "MODEL_NAME": "SSD mobilenet v1 with COCO 2014 dataset",
```
Comment (psyhtest) [inline package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json]: SSD-MobileNet-v1 for COCO?

File: package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json [hunk 8]
```diff
@@ -3,52 +3,63 @@
...
-  },
-  "suggested_path": "tensorflowapimodel-ssd-mobilenet-v1-coco",
+  }, 
+  "only_for_host_os_tags": [
+    "linux"
+  ], 
+  "only_for_target_os_tags": [
+    "linux"
+  ], 
+  "package_extra_name": " (SSD Mobilenet v1, coco)",
```
Comment (psyhtest) [inline package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json]: SSD-MobileNet-v1, COCO

File: package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json [hunk 9]
```diff
@@ -3,52 +3,63 @@
...
-    "ssd",
-    "mobilenet",
-    "mobilenetv1",
-    "coco",
+    "tensorflow", 
+    "model", 
+    "tensorflowapi", 
+    "tensorflowapimodel", 
+    "ssd", 
+    "mobilenet",
```
Comment (psyhtest) [inline package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json]: ssd-mobilenet, ssd-mobilenet-v1

File: package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json [hunk 10]
```diff
@@ -3,52 +3,63 @@
...
-    "mobilenet",
-    "mobilenetv1",
-    "coco",
+    "tensorflow", 
+    "model", 
+    "tensorflowapi", 
+    "tensorflowapimodel", 
+    "ssd", 
+    "mobilenet", 
+    "mobilenet_v1",
```
Comment (psyhtest) [inline package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json]: mobilenet-v1?

File: package/tensorflowmodel-api/README.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
+## Installation
+
+**Attention!!!** This package is intended for Linux only
+
+You should have `ck-tensorflow` repository installed:
+```bash
+$ cd ~/CK
+$ ck pull repo --url=https://github.com/dividiti/ck-tensorflow
```
Comment (psyhtest) [inline package/tensorflowmodel-api/README.md]:
```
$ ck pull repo:ck-tensorflow
```

File: program/tensorflow-api/.cm/meta.json [hunk 12]
```diff
@@ -6,16 +6,34 @@
...
+    },
+    "dataset": {
+      "local": "yes",
+      "sort": 20,
+      "name": "Object-detection dataset", 
+      "tags": "dataset,object-detection"
+    },
+    "tensorflow-api-model":{
+      "local": "yes",
+      "name": "TensorFlow Object recognition API model",
```
Comment (psyhtest) [inline program/tensorflow-api/.cm/meta.json]: TensorFlow Object Detection API model

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/scripts.linux/install.sh (hunk 5 above)
Comment (psyhtest) [reply package/tensorflowapimodel-faster-rcnn-resnet101-kitti/scripts.linux/install.sh]: (Please also change elsewhere.)

File: program/tensorflow-api/.cm/meta.json (hunk 12 above)
Comment (bellycat77) [reply program/tensorflow-api/.cm/meta.json]: Done

File: package/tensorflowmodel-api/README.md (hunk 11 above)
Comment (bellycat77) [reply package/tensorflowmodel-api/README.md]: done

File: package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json (hunk 10 above)
Comment (bellycat77) [reply package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json]: Done

File: package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json (hunk 9 above)
Comment (bellycat77) [reply package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json]: Done

File: package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json (hunk 8 above)
Comment (bellycat77) [reply package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json]: Done

File: package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json (hunk 7 above)
Comment (bellycat77) [reply package/tensorflowapimodel-ssd-mobilenet-v1-coco/.cm/meta.json]: Done

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/kitti_label_map.pbtxt (hunk 6 above)
Comment (bellycat77) [reply package/tensorflowapimodel-faster-rcnn-resnet101-kitti/kitti_label_map.pbtxt]: This model don't know anything  about 'cyclists' - just cars and pedestrians

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/scripts.linux/install.sh (hunk 5 above)
Comment (bellycat77) [reply package/tensorflowapimodel-faster-rcnn-resnet101-kitti/scripts.linux/install.sh]: Done

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json (hunk 3 above)
Comment (bellycat77) [reply package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json]: Done

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json (hunk 2 above)
Comment (bellycat77) [reply package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json]: Done

File: package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json (hunk 1 above)
Comment (bellycat77) [reply package/tensorflowapimodel-faster-rcnn-resnet101-kitti/.cm/meta.json]: Done

**Pull request merged**